README.md

```diff
@@ -652,6 +653,42 @@ Creates a version of the function that will only be run after first being called
  --- | --- | --- | --- | --- |
    ✔  |  ✔ |  ✔ |  ✔ |  ✔  |  
  
+ ## _.assign
+The method is used to copy the values of all enumerable own properties from one or more source objects to a target object.
+
+  ```js
+  // Underscore/Lodash
```

Comment:
Hi @plantain-00 , I think Underscore doesn't have _.assign method. So this could be a Lodash only function. Could you change it? ( Follow the format of Lodash only functions)